templates/code-of-conduct.md

```diff
@@ -0,0 +1,5 @@
+# Code of Conduct
+
+Please refer to the [CHAOSS Community Code of Conduct](https://github.com/chaoss/governance/blob/master/code-of-conduct.md)
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the CHAOSS Code of Conduct Committee via  <chaoss-inclusion@lists.linuxfoundation.org>
```

Comment:
```suggestion
Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the CHAOSS Code of Conduct Committee via email: <chaoss-inclusion@lists.linuxfoundation.org>
```